h/resources.py

```diff
@@ -134,6 +134,38 @@ def __getitem__(self, pubid):
         return organization.logo
 
 
+class GroupResource(object):
+    def __init__(self, group, links_service):
+        self.group = group
+        self.links_service = links_service
+
+    @property
+    def links(self):
+        return self.links_service.get_all(self.group)
+
+
+class OrganizationResource(object):
+    def __init__(self, organization, links_service=None):
+        # TBD: Links service for org links
+        self.organization = organization
+        self.links_service = links_service
+
+    @property
+    def links(self):
+        return {}
+
+    @property
+    def logo(self):
+        return ''
+
+
+class ExpandedGroupResource(GroupResource):
```

Comment:
This is one possible approach. "Expanded" objects would have certain relations expanded into the Resource representation of the target model...

As an alternative, these expansions could happen by default in `GroupResource`, but it seemed like not the greatest from a performance perspective?

I feel like there are probably a lot of ways to skin this cat.